api/src/test/java/org/pmiops/workbench/google/CloudStorageClientTest.java

```diff
@@ -0,0 +1,60 @@
+package org.pmiops.workbench.google;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.google.cloud.storage.Blob;
+import java.time.Clock;
+import java.time.Instant;
+import java.time.ZoneId;
+import org.junit.jupiter.api.Test;
+import org.pmiops.workbench.model.FileDetail;
+import org.pmiops.workbench.test.FakeClock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
+
+@DataJpaTest
+public class CloudStorageClientTest {
+
+  private static final FakeClock CLOCK = new FakeClock(Instant.now(), ZoneId.systemDefault());
+
+  @Autowired private CloudStorageClient cloudStorageClient;
+
+  @TestConfiguration
+  @Import({
+    CloudStorageClientImpl.class
+  })
+
+  static class Configuration {
+
+    @Bean
+    Clock clock() {
+      return CLOCK;
+    }
+  }
+
+  @Test
+  public void getExtractionJobs() {
```

Comment:
nice test.  Not sure about this name though - copy/paste?

Reply:
Whoops. Thanks.